### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by bigjoevtrj** [kaisermann/svelte-i18n#7]
Any imports leads to:
```
Uncaught ReferenceError: o is not defined
```
main.js
```
import './lang.js';
```

lang.js
```
import { locale, dictionary } from 'svelte-i18n';
locale.set('pt');
```

**Comment by bigjoevtrj** [comment]
intl-format-cache is not installing automatically, that's the reason to this error.

**Comment by kaisermann** [comment]
@bigjoevtrj Ooops, my bad! I published the package without rebuilding it. Fixed on `v1.0.3` 🙏 

Thanks!
